documentation/host/verification-stages.mdx

```diff
@@ -1,110 +1,161 @@
       },
       {
         "@type": "HowToStep",
         "name": "Meet Verification Requirements",
-        "text": "Ensure your machine meets verification requirements: CUDA version greater than or equal to 12.0, reliability of 90%, at least 3 open ports per GPU (100 recommended), internet download speed of 500 Mb/s, internet upload speed of 500 Mb/s, and GPU RAM of 7 GB."
+        "text": "Satisfy additional verification requirements: CUDA version 12.0 or higher, maintain 90%+ reliability, provide at least 3 open ports per GPU (100 recommended), achieve 500+ Mb/s internet speeds (both download and upload), and ensure GPUs have at least 7GB VRAM."
       },
       {
         "@type": "HowToStep",
-        "name": "Run Self-Test",
-        "text": "Use the vast cli self-test machine command to see if a machine meets all of the requirements and resolve any issues. This will give you a better chance of getting verified."
+        "name": "Maintain Host Responsibilities",
+        "text": "Keep systems stable, well-cooled, and correctly configured. Maintain compatible drivers/CUDA and dependable symmetric networking. Run only platform jobs via Create Job. When issues arise, fix them promptly as the automation will update verification status accordingly."
       },
       {
         "@type": "HowToStep",
-        "name": "Wait for Verification",
-        "text": "Verification is mostly for higher end machines. High-end machines ($10k+) like H100 or A100 80GB should be tested quickly in a day or so. 8x4090, 4xA6000 should be tested in less than a week. For other machines, verification is relatively random and runs roughly about once a week. The only manual verification tests are for datacenters and high end machines."
+        "name": "Optimize During Unverified Stage",
+        "text": "While unverified, maintain steady uptime during evaluation. Ensure drivers/CUDA and networking are correctly installed and reachable. Keep environment clean by scheduling work via Create Job only. Avoid reboots, configuration changes, and unrelated background workloads."
       },
       {
         "@type": "HowToStep",
-        "name": "Maintain Verification Status",
-        "text": "Once verified, continue to meet requirements and prevent issues that could get you deverified. Watch for red errors on your machine card and investigate and resolve them. Issues with container starting, nvidia-smi related errors, nvml errors, nvidia-container-cli device errors, or bandwidth verification issues could get your machine deverified. If deverified and the issue is resolved, your machine will go from deverified back to unverified."
+        "name": "Maintain Verified Status",
+        "text": "Once verified, monitor health including uptime, thermals, and power, responding to alerts promptly. Keep drivers/CUDA on compatible latest stable versions. Maintain stable symmetric bandwidth. Avoid downgrading hardware capacity or allowing thermal, power, or bandwidth instability."
```

Comment:
"Once verified, be sure to continue monitoring the health of your machine, including uptime, thermals, and power. You should respond to any alerts promptly. Keep drivers/CUDA on compatible latest stable versions. Maintain stable symmetric bandwidth. Avoid downgrading hardware capacity or allowing thermal, power, or bandwidth instability."